doc/htmldoc/connect_nest/nest_server.rst

```diff
@@ -148,21 +147,21 @@ completely the same as one coming from the serial version of the NEST
 Server. The only difference may be that information pertaining to
 process-local data structures is being replaced by generic values.
 
-.. _nest_server_client:
+.. _nest_client:
 
-The NEST Server Client
-----------------------
+The NEST Client
+---------------
 
-The easiest way to interact with the NEST Server is the `NEST Server
-Client` provided in ``examples/NESTServerClient`` in the source
-distribution of NEST. It can be used either by directly starting
-a Python session in that directory or installing it by running ``python3
-setup.py install`` therein. NEST itself does not have to be installed
-in order to use the NEST Server Client.
+The easiest way to interact with the NEST Server is the `NEST Client`
+provided in <https://github.com/nest/nest-client/>`_. It can be used
+either by directly starting a Python session in a clone of that
+repository, or by installing it by running ``python3 setup.py
+install`` therein. NEST itself does not have to be installed in order
+to use the NEST Client.
```

Comment:
```suggestion
The easiest way to interact with the NEST Server is the `NEST Client`
provided in `<https://github.com/nest/nest-client/>`_. It can be used
either by directly starting a Python session in a clone of that
repository, or by installing it by running ``python3 setup.py
install`` therein. NEST itself does not have to be installed in order
to use the NEST Client.
```

Reply:
Nice catch. Thanks!